planner/core/testdata/plan_suite_in.json

```diff
@@ -540,5 +540,20 @@
       "select lead(a, 1) over (partition by null) as c from t",
       "select lead(a, 1) over (partition by b) as c from t"
     ]
+  },
+  {
+    "name": "TestNominalSort",
```

Comment:
We only have `explain` results here. Could you add some test cases that will generate row result instead of the query plan?